Account/Quotas/Program.cs

```diff
@@ -4,52 +4,113 @@
 using Azure.Identity;
 using Azure.Monitor.Query;
 using Azure.ResourceManager.Media;
+using Microsoft.Extensions.Configuration;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
 
-var quotas = new QuotaMetrics[]
+internal class Program
```

Comment:
Why is this being wrapped in a class?

Reply:
I find it clearer that sample code without a class.

Reply:
For these simple sample applications, wrapping the code in a class (and sometimes a namespace), doesn't add any value.

Reply:
Ok, I removed them